editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings.tsx

```diff
@@ -102,6 +113,36 @@ const ServiceSettings: React.FC = () => {
     setIsAlertOpen(false);
   };
 
+  const sendFlowStatusSlackNotification = async (status: FlowStatus) => {
+    const skipTeamSlugs = [
+      "open-digital-planning",
+      "opensystemslab",
+      "planx",
+      "templates",
+      "testing",
+      "wikihouse",
+    ];
+    if (skipTeamSlugs.includes(teamSlug)) return;
```

Comment:
The API handles "environment" logic because that should consistently apply to any scenario we're using this endpoint for, but the "skipable teams" logic is specific to flow status notifications, so direclty handled here. 

Open to alternate suggestions !